src/test/ui/impl-trait/auto-trait-leak.stderr

```diff
@@ -0,0 +1,52 @@
+error[E0277]: the trait bound `std::rc::Rc<std::cell::Cell<i32>>: std::marker::Send` is not satisfied in `impl std::ops::Fn<(i32,)>`
+  --> $DIR/auto-trait-leak.rs:27:5
+   |
+27 |     send(before());
+   |     ^^^^ `std::rc::Rc<std::cell::Cell<i32>>` cannot be sent between threads safely
+   |
+   = help: within `impl std::ops::Fn<(i32,)>`, the trait `std::marker::Send` is not implemented for `std::rc::Rc<std::cell::Cell<i32>>`
+   = note: required because it appears within the type `[closure@$DIR/auto-trait-leak.rs:21:5: 21:22 p:std::rc::Rc<std::cell::Cell<i32>>]`
+   = note: required because it appears within the type `impl std::ops::Fn<(i32,)>`
+   = note: required by `send`
+
+error[E0277]: the trait bound `std::rc::Rc<std::cell::Cell<i32>>: std::marker::Send` is not satisfied in `impl std::ops::Fn<(i32,)>`
+  --> $DIR/auto-trait-leak.rs:34:5
+   |
+34 |     send(after());
+   |     ^^^^ `std::rc::Rc<std::cell::Cell<i32>>` cannot be sent between threads safely
+   |
+   = help: within `impl std::ops::Fn<(i32,)>`, the trait `std::marker::Send` is not implemented for `std::rc::Rc<std::cell::Cell<i32>>`
+   = note: required because it appears within the type `[closure@$DIR/auto-trait-leak.rs:46:5: 46:22 p:std::rc::Rc<std::cell::Cell<i32>>]`
+   = note: required because it appears within the type `impl std::ops::Fn<(i32,)>`
+   = note: required by `send`
+
+error[E0391]: unsupported cyclic reference between types/traits detected
+  --> $DIR/auto-trait-leak.rs:52:1
+   |
+52 | fn cycle1() -> impl Clone {
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^ cyclic reference
+   |
+note: the cycle begins when processing `cycle1`...
+  --> $DIR/auto-trait-leak.rs:52:1
+   |
+52 | fn cycle1() -> impl Clone {
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Actual output differs from expected output. Please update.

```diff
 error[E0391]: unsupported cyclic reference between types/traits detected
   --> $DIR/auto-trait-leak.rs:52:1
    |
-52 | fn cycle1() -> impl Clone {
-   | ^^^^^^^^^^^^^^^^^^^^^^^^^ cyclic reference
+52 |   fn cycle1() -> impl Clone {
+   |  _-^^^^^^^^^^^^^^^^^^^^^^^^
+53 | |     //~^ ERROR unsupported cyclic reference between types/traits detected
+54 | |     //~| cyclic reference
+55 | |     //~| NOTE the cycle begins when processing `cycle1`...
+...  |
+60 | |     Rc::new(Cell::new(5))
+61 | | }
+   | |_- cyclic reference
```

Reply:
Missed the label. Done.